src/components/breadcrumb/index.njk

```diff
@@ -8,44 +8,7 @@ A breadcrumb is a GOV.UK element that helps users to understand where they are w
 {% endblock %}
 
 {% block defaultAndVariants %}
-{% for item in componentData.variants %}
-{% if item.name == 'default' %}
-  {% set itemName = 'Component default' %}
-  {% set previewLink = "/components/" + componentName + '/' + "/preview" %}
-  {% set previewText = 'Preview the ' + componentName + ' component' %}
-{% else %}
-  {% set itemName = componentName + '--' + item.name %}
-  {% set previewLink = "/components/" + componentName + '/' + componentName + '--' + item.name  + "/preview" %}
-  {% set previewText = 'Preview the ' + itemName + ' variant' %}
-{% endif %}
-
-<h3 class="govuk-u-bold-19">{{ itemName  | capitalize }}</h3>
-
-{% if not isReadme %}
-{{ govukBreadcrumb(item.data) }}
-
-
-<p class="govuk-u-copy-19">
-<a href="{% if isReadme %}http://govuk-frontend-review.herokuapp.com{% endif %}{{ previewLink }}">{{previewText}}
-</a>
-</p>
-{% endif %}
-<h4 class="govuk-u-copy-19">Markup</h4>
-
-{% set componentHtml %}
-{{ govukBreadcrumb(item.data) }}
-{% endset %}
-<pre><code>
-  {{- componentHtml | e -}}
-</code></pre>
-
-<h4 class="govuk-u-copy-19">Macro</h4>
-<pre><code>
-{% raw %}{{ govukBreadcrumb({% endraw %}
-  {{ item.data | dump(2) }}
-{% raw %}) }}{% endraw %}
-</code></pre>
-{% endfor %}
+{{ showDefaultAndVariants(componentName, componentData) }}
```

Comment:
Given this appears to be the same across all components, can this now just go into the `defaultAndVariants` block in `component.njk`?

Reply:
Fair point. I'll move it inside.